src/univalue/include/univalue.h

```diff
@@ -20,10 +20,7 @@ class UniValue {
     enum VType { VNULL, VOBJ, VARR, VSTR, VNUM, VBOOL, };
 
     UniValue() { typ = VNULL; }
-    UniValue(UniValue::VType initialType, const std::string& initialStr = "") {
-        typ = initialType;
-        val = initialStr;
-    }
+    UniValue(UniValue::VType type, std::string str = "") : typ{type}, val{std::move(str)} {}
```

Comment:
In commit "univalue: Avoid std::string copies" (fa1ce3ab42fd5fd761b934b8db9fbf9f78bde842)

Probably doesn't matter in practice, but it seems like it could be less work for the compiler to call the default string constructor instead of the `char*` constructor

```diff
diff --git a/src/univalue/include/univalue.h b/src/univalue/include/univalue.h
index 54c60cada9a..6ef60b16e29 100644
--- a/src/univalue/include/univalue.h
+++ b/src/univalue/include/univalue.h
@@ -20,7 +20,7 @@ public:
     enum VType { VNULL, VOBJ, VARR, VSTR, VNUM, VBOOL, };
 
     UniValue() { typ = VNULL; }
-    UniValue(UniValue::VType type, std::string str = "") : typ{type}, val{std::move(str)} {}
+    UniValue(UniValue::VType type, std::string str = {}) : typ{type}, val{std::move(str)} {}
     template <typename Ref, typename T = std::remove_cv_t<std::remove_reference_t<Ref>>,
               std::enable_if_t<std::is_floating_point_v<T> ||                      // setFloat
                                    std::is_same_v<bool, T> ||                      // setBool
```

Reply:
Thx, done